front/lib/api/assistant/conversation.ts

```diff
@@ -499,6 +490,17 @@ export async function* postUserMessage(
         context: context,
       };
 
+      if (user) {
```

Comment:
Should be done concurrently to the Message.create call above?

Also should be created only if it does not exist?

Reply:
If we want to use it at some point for a notification system we'll have to have the last date when an action occurred, but at the moment indeed it makes way more sense to create only if not exist. Adding a commit!